src/test/resources/plugin-security-test.policy

```diff
@@ -0,0 +1,6 @@
+grant {
+  permission java.net.SocketPermission "localhost:1024-", "listen,resolve,connect,accept";
```

Comment:
`connect` and `accept` permissions should also be added to [plugin-security.policy](https://github.com/Automattic/elasticsearch-statsd-plugin/blob/master/src/plugin-metadata/plugin-security.policy#L2) or ES fails to start with :

```[2018-01-06T13:00:46,782][WARN ][o.e.b.ElasticsearchUncaughtExceptionHandler] [node-1] uncaught exception in thread [main]
org.elasticsearch.bootstrap.StartupException: org.elasticsearch.common.inject.ProvisionException: Guice provision errors:

1) Error injecting constructor, com.timgroup.statsd.StatsDClientException: Failed to start StatsD client
  at com.automattic.elasticsearch.statsd.StatsdService.<init>(Unknown Source)
  while locating com.automattic.elasticsearch.statsd.StatsdService

1 error
	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:125) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Elasticsearch.execute(Elasticsearch.java:112) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:86) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:124) ~[elasticsearch-cli-6.1.1.jar:6.1.1]
	at org.elasticsearch.cli.Command.main(Command.java:90) ~[elasticsearch-cli-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:92) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:85) ~[elasticsearch-6.1.1.jar:6.1.1]
Caused by: org.elasticsearch.common.inject.ProvisionException: Guice provision errors:

1) Error injecting constructor, com.timgroup.statsd.StatsDClientException: Failed to start StatsD client
  at com.automattic.elasticsearch.statsd.StatsdService.<init>(Unknown Source)
  while locating com.automattic.elasticsearch.statsd.StatsdService

1 error
	at org.elasticsearch.common.inject.InjectorImpl$4.get(InjectorImpl.java:771) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.common.inject.InjectorImpl.getInstance(InjectorImpl.java:801) ~[elasticsearch-6.1.1.jar:6.1.1]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193) ~[?:1.8.0_121]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1374) ~[?:1.8.0_121]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481) ~[?:1.8.0_121]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471) ~[?:1.8.0_121]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708) ~[?:1.8.0_121]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_121]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499) ~[?:1.8.0_121]
	at org.elasticsearch.node.Node.<init>(Node.java:508) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.node.Node.<init>(Node.java:245) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:212) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:212) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:322) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:121) ~[elasticsearch-6.1.1.jar:6.1.1]
	... 6 more
Caused by: com.timgroup.statsd.StatsDClientException: Failed to start StatsD client
	at com.timgroup.statsd.NonBlockingStatsDClient.<init>(NonBlockingStatsDClient.java:115) ~[?:?]
	at com.timgroup.statsd.NonBlockingStatsDClient.<init>(NonBlockingStatsDClient.java:82) ~[?:?]
	at com.automattic.elasticsearch.statsd.StatsdService$1.run(StatsdService.java:75) ~[?:?]
	at com.automattic.elasticsearch.statsd.StatsdService$1.run(StatsdService.java:72) ~[?:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_121]
	at com.automattic.elasticsearch.statsd.StatsdService.<init>(StatsdService.java:72) ~[?:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:?]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_121]
	at org.elasticsearch.common.inject.DefaultConstructionProxyFactory$1.newInstance(DefaultConstructionProxyFactory.java:49) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.common.inject.ConstructorInjector.construct(ConstructorInjector.java:86) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.common.inject.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:116) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.common.inject.InjectorImpl$4$1.call(InjectorImpl.java:762) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.common.inject.InjectorImpl.callInContext(InjectorImpl.java:818) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.common.inject.InjectorImpl$4.get(InjectorImpl.java:757) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.common.inject.InjectorImpl.getInstance(InjectorImpl.java:801) ~[elasticsearch-6.1.1.jar:6.1.1]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193) ~[?:1.8.0_121]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1374) ~[?:1.8.0_121]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481) ~[?:1.8.0_121]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471) ~[?:1.8.0_121]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708) ~[?:1.8.0_121]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_121]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499) ~[?:1.8.0_121]
	at org.elasticsearch.node.Node.<init>(Node.java:508) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.node.Node.<init>(Node.java:245) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:212) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:212) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:322) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:121) ~[elasticsearch-6.1.1.jar:6.1.1]
	... 6 more
Caused by: java.security.AccessControlException: access denied ("java.net.SocketPermission" "localhost:0" "listen,resolve")
	at java.security.AccessControlContext.checkPermission(AccessControlContext.java:472) ~[?:1.8.0_121]
	at java.security.AccessController.checkPermission(AccessController.java:884) ~[?:1.8.0_121]
	at java.lang.SecurityManager.checkPermission(SecurityManager.java:549) ~[?:1.8.0_121]
	at java.lang.SecurityManager.checkListen(SecurityManager.java:1131) ~[?:1.8.0_121]
	at java.net.DatagramSocket.bind(DatagramSocket.java:389) ~[?:1.8.0_121]
	at java.net.DatagramSocket.<init>(DatagramSocket.java:242) ~[?:1.8.0_121]
	at java.net.DatagramSocket.<init>(DatagramSocket.java:196) ~[?:1.8.0_121]
	at com.timgroup.statsd.NonBlockingStatsDClient.<init>(NonBlockingStatsDClient.java:112) ~[?:?]
	at com.timgroup.statsd.NonBlockingStatsDClient.<init>(NonBlockingStatsDClient.java:82) ~[?:?]
	at com.automattic.elasticsearch.statsd.StatsdService$1.run(StatsdService.java:75) ~[?:?]
	at com.automattic.elasticsearch.statsd.StatsdService$1.run(StatsdService.java:72) ~[?:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_121]
	at com.automattic.elasticsearch.statsd.StatsdService.<init>(StatsdService.java:72) ~[?:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:?]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_121]
	at org.elasticsearch.common.inject.DefaultConstructionProxyFactory$1.newInstance(DefaultConstructionProxyFactory.java:49) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.common.inject.ConstructorInjector.construct(ConstructorInjector.java:86) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.common.inject.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:116) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.common.inject.InjectorImpl$4$1.call(InjectorImpl.java:762) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.common.inject.InjectorImpl.callInContext(InjectorImpl.java:818) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.common.inject.InjectorImpl$4.get(InjectorImpl.java:757) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.common.inject.InjectorImpl.getInstance(InjectorImpl.java:801) ~[elasticsearch-6.1.1.jar:6.1.1]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193) ~[?:1.8.0_121]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1374) ~[?:1.8.0_121]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481) ~[?:1.8.0_121]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471) ~[?:1.8.0_121]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708) ~[?:1.8.0_121]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_121]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499) ~[?:1.8.0_121]
	at org.elasticsearch.node.Node.<init>(Node.java:508) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.node.Node.<init>(Node.java:245) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:212) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:212) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:322) ~[elasticsearch-6.1.1.jar:6.1.1]
	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:121) ~[elasticsearch-6.1.1.jar:6.1.1]
	... 6 more
```